sem_temp.html

```diff
@@ -0,0 +1,188 @@
+    />
+    <meta
+      name="summary"
+      content="An open source project aimed at providing free and beautiful templates to everyone for building their portfolio websites and showcase their work to the world."
+    />
+    <link
+      rel="stylesheet"
+      href="https://maxcdn.bootstrapcdn.com/bootstrap/4.5.0/css/bootstrap.min.css"
+    />
+
+    <!--For Dark/Light mode Toggle-->
+    <script src="https://code.iconify.design/1/1.0.4/iconify.min.js"></script>
+
+    <!-- Loading external stylesheets -->
+    <link
+      href="https://fonts.googleapis.com/css2?family=Poppins:wght@200;300&display=swap"
+      rel="stylesheet"
+    />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/animate.css/4.0.0/animate.min.css"
+    />
+    <link href="https://unpkg.com/aos@2.3.1/dist/aos.css" rel="stylesheet" />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"
+      type="text/css"
+    />
+
+    <!-- Font Awesome Script -->
+    <script src="https://kit.fontawesome.com/a076d05399.js"></script>
+    <link rel="stylesheet" href="assets/css/hover-min.css" />
+    <script src="https://code.iconify.design/1/1.0.7/iconify.min.js"></script>
+
+    <!-- Preloader stylesheet -->
+    <link rel="stylesheet" href="assets/css/preloader.css" />
+    <link rel="stylesheet" href="assets/css/sem_temp.css" />
+    <link rel="stylesheet" href="assets/css/style.css" />
+    <link rel="stylesheet" href="assets/css/cursor.css" />
+
+    <!-- Favicon -->
+    <link
+      id="favicon"
+      rel="icon"
+      href="assets/images/favicon.png"
+      type="image/png"
+    />
+
+    <script defer src="./assets/js/dynamicTitle.js"></script>
+  </head>
+
+  <body class="no-scroll-preload">
+    <!-- loader -->
+    <div class="loader-container">
+      <div class="atom">
+        <div class="electron"></div>
+        <div class="electron-alpha"></div>
+        <div class="electron-omega"></div>
+      </div>
+    </div>
+
+    <!--Dynamic Nav Bar-->
+
+    <div class="untitled">
+      <div class="untitled__slides">
+        <div class="untitled__slide">
+          <div class="untitled__slideBg"></div>
+          <div class="untitled__slideContent">
+            <span>Chemistry Lab</span>
+            <span>Mentorship</span>
+          </div>
+        </div>
+
+        <div class="untitled__slide">
+          <div class="untitled__slideBg"></div>
+          <div class="untitled__slideContent">
+            <span>User Experience</span>
+            <span>Summer Research</span>
+          </div>
+        </div>
+
+        <div class="untitled__slide">
+          <div class="untitled__slideBg"></div>
+          <div class="untitled__slideContent">
+            <span>UX Team</span>
+            <span>Happy Hours</span>
+          </div>
+        </div>
+
+        <div class="untitled__slide">
+          <div class="untitled__slideBg"></div>
```

Comment:
We can have this loaded from a sem_temp.js file where the title and the images are defined inside a JSON variable and used in a loop. (Like we have it for the project or experience pages)